pyxrf/xanes_maps/xanes_maps_api.py

```diff
@@ -583,6 +600,9 @@ def _compute_xanes_maps(*, start_id, end_id, wd_xrf,
 
     logger.info("Sorting dataset: success.")
 
+    # Apply shift to scan energies
+    scan_energies_shifted = [_ + incident_energy_shift_keV for _ in scan_energies]
```

Comment:
Could this be done with numpy?
```
scan_energies_shifted = scan_energies + incident_energy_shift_keV
```

Reply:
``scan_energies`` is a list (below 100 elements). I don't think that moving to ndarray is worth it.

Reply:
I don't think the size should matter here. 3x3 arrays/matrices (as an irrelevant example to this case) are also relatively simple and small, but using them as ndarrays are much more pleasant. I think it costs nothing to use that list as a numpy array here as the library is already imported and used.

Reply:
My idea was to keep all parameters of individual scans as lists, one of those parameters is ``scan_energies``. Shifting the energy axis was part of the latest request, so I decided to keep shifted energy values in the list as well, so that the data structures remain consistent. At this point, adding a constant to every element of the list is the only operation that is performed.